testing/unittests/CMakeLists.txt

```diff
@@ -42,8 +44,9 @@ set (    SRCS
          txSummarytests.cpp
          accessmaptests.cpp
          fxpptests.cpp
-         ${HPCC_SOURCE_DIR}/esp/bindings/SOAP/xpp/fxpp/FragmentedXmlPullParser.cpp;
-         ${HPCC_SOURCE_DIR}/esp/bindings/SOAP/xpp/fxpp/FragmentedXmlAssistant.cpp;
+         ${HPCC_SOURCE_DIR}/esp/bindings/SOAP/xpp/fxpp/FragmentedXmlPullParser.cpp
+         ${HPCC_SOURCE_DIR}/esp/bindings/SOAP/xpp/fxpp/FragmentedXmlAssistant.cpp
+         ${CMAKE_BINARY_DIR}/generated/ws_loggingservice_esp.cpp
```

Comment:
@afishbeck - please review + see question re. this and generated export definition in JIRA.

Reply:
Regarding question in JIRA: "yes I think those factories should all have the visibility attributes added. Some of those functions are rarely used externally so probably slipped through the cracks. The most common external usage is to go through the client interface. But some of these testing and configuration uses go directly to some of the less common objects."